### DISCUSSION TITLE
How to force build libCppUTest.a Static Library

### DISCUSSION THREAD

**Discussion by DBizz22** [cpputest/cpputest#1816] [Q&A] [answered]
Hello, Goodday. I am new to using CppUTest. I ran into an issue of the configuration building a dynamic library libCppUTest.dll which makes cpputest_discover_tests fail to discover the tests when added as one of the external Project using Fetchcontent to my project after being linked to the tests executable. My question is, how do i force the configuration to build a libCppuTest.a Static library which when linked to the test target will work?

**Comment by thetic** [comment, marked as answer]
CMake will build libraries as `STATIC` by default unless [`BUILD_SHARED_LIBS`](https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html) is set.

**Reply by DBizz22** [reply]
Thank you very much
